src/command.ts

```diff
@@ -199,14 +199,24 @@ export default abstract class Command {
     return Errors.error(input, options as any)
   }
 
-  log(message = '', ...args: any[]): void {
+  stdout(message = '', ...args: any[]): void {
     if (!this.jsonEnabled()) {
-      // tslint:disable-next-line strict-type-predicates
       message = typeof message === 'string' ? message : inspect(message)
       process.stdout.write(format(message, ...args) + '\n')
     }
   }
 
+  stderr(message = '', ...args: any[]): void {
```

Comment:
Similarly to my comment above I'd rather not name this `stderr`. Perhaps we can go with use `error()` or `logError()`?

Reply:
I don't think using `logError()` and the implicit assumption of only logging errors to `stderr` is necessarily a bad thing. It's the same assumption the POSIX standard makes by naming that stream `stderr` instead of `nonstdout` or something that encompasses more than just errors.

If users want to use `stderr` for something other than errors that's up to them.

That being said, I'm fine with using `logToStderr` if you'd rather do that since my whole point here is just to make it clear this method is something different than `process.stderr` and `logToStderr` achieves that.